apps/mobile/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,74 @@
+import CheckBox from "@react-native-community/checkbox";
+import React from "react";
+
+import { LinearGradient, Text, View } from "~/libs/components/components";
+import { BaseColor, GradientColor, GradientPoint } from "~/libs/enums/enums";
+import { globalStyles } from "~/libs/styles/global-styles/global-styles";
+
+import { styles } from "./styles";
+
+type Properties = {
+	isChecked: boolean;
+	label?: string;
```

Comment:
`label` is mandatory. Add `hasVisuallyHiddenLabel` prop and if it is true, hide the label